mern/client/src/utils/QA.js

```diff
@@ -1,21 +1,35 @@
 import axios from "axios";
 
+var allQuestionList = [];
+
 export default class Parser {
   parse(transcript) {
-    const lst = [];
+    const startingList = [];
+
+    if (allQuestionList !== []) {
```

Comment:
This is because starting list is a "global" variable. This was the reason why every time we uploaded a transcript after deleting it, the starting intents list doubled in size. The function just keeps pushing without resetting to 0.